### PR TITLE
Broadcast manager UI

### PR DESCRIPTION
**Pull request by shadyvb**
Ref https://github.com/humanmade/product-dev/issues/1207

### CODE REVIEW

File: src/data/index.ts [hunk 1]
```diff
@@ -207,6 +219,18 @@ const actions = {
 			isLoading,
 		};
 	},
+	/**
+	 * Set is updating action creator.
+	 *
+	 * @param {boolean} isUpdating True if UI is updating data.
+	 * @returns {object} Set updating action object.
+	 */
+	setIsUpdating ( isUpdating: boolean ): SetIsUpdatingAction {
```
Comment (roborourke) [inline src/data/index.ts:228]: Some inconsistent code style here, is that from an updated rule set?
```suggestion
	setIsUpdating( isUpdating: boolean ) : SetIsUpdatingAction {
```

File: src/data/index.ts [hunk 2]
```diff
@@ -340,11 +475,37 @@ const resolvers = {
...
+			yield actions.setPagination(
+				parseInt( response.headers.get( 'x-wp-total' ) || '0', 10 ),
+				parseInt( response.headers.get( 'x-wp-totalpages' ) || '0', 10 )
+			);
+			return actions.setIsLoading( false );
+		} else {
+			return posts;
+		}
+	},
+	*getPost ( id: number, type: string ) {
```
Comment (roborourke) [inline src/data/index.ts:489]: Missing docblock

File: src/accelerate/broadcast-manager/index.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+				period={ period }
+				setPeriod={ setPeriod }
+				filters={ [ 'all', 'mine' ] }
+				onAddNewItem={ () => setAddingNewItem( true ) }
+				onManageItem={ setManagingItem }
+				postId={ broadcastId }
+			/>
+			{ addingNewItem && (
+				<CreateModal
+					listId='Broadcast Manager'
```
[the remark this answers is not in the document]
Comment (roborourke) [reply src/accelerate/broadcast-manager/index.tsx:39]: Can see it's for the tracking - no worries then :)

File: src/data/index.ts [hunk 4]
```diff
@@ -132,6 +132,18 @@ const actions = {
 			key,
 		};
 	},
+	/**
+	 * Update a post entry.
+	 *
+	 * @param post Updated post entry.
+	 * @returns {object} Update post action object.
+	 */
+	setPost ( post: Post ): SetPostAction {
```
Comment (shadyvb) [inline src/data/index.ts:141]:
```suggestion
	setPost( post: Post ) : SetPostAction {
```

File: src/accelerate/broadcast-manager/modals/Create.tsx [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+					label={ __( 'Choose a name for the new Broadcast:', 'altis' ) }
+					onChange={ setTitle }
+					value={ title }
+					autoFocus
+					disabled={ isSaving }
+				/>
+
+				<ButtonGroup>
+					<Button isPrimary disabled={ isSaving } onClick={ onSave }>
+						{ isSaving ? __( 'Creating..', 'altis' ) : __( 'Create', 'altis' ) }
```
[the remark this answers is not in the document]
Comment (shadyvb) [reply src/accelerate/broadcast-manager/modals/Create.tsx]: TIL!

**Pull request merged**